src/solvers/mp/RCPricer.cpp

```diff
@@ -63,24 +63,22 @@ vector<MyVar*> RotationPricer::pricing(double bound, bool before_fathom){
 	double minDualCost = 0;
 	vector<LiveNurse*> nursesSolved;
 
-	for(vector<LiveNurse*>::iterator it0 = nursesToSolve_.begin(); it0 != nursesToSolve_.end();){
+	for(auto it0 = nursesToSolve_.begin(); it0 != nursesToSolve_.end();){
```

Comment:
Just thinking out loud: when we were talking about parallelism, it was about this loop I guess. But is it not more critical to parallelize the exploration of the branching tree? (e.g. multiple dives)
 Anyways, let's create an issue for parallelism?

Reply:
ok :)